fluster/platformdirs.py

```diff
@@ -0,0 +1,64 @@
+# Fluster - testing framework for decoders conformance
+# Copyright (C) 2023, Fluendo, S.A.
+#  Author: Ruben Gonzalez <rgonzalez@fluendo.com>, Fluendo, S.A.
+#
+# This library is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public License
+# as published by the Free Software Foundation, either version 3
+# of the License, or (at your option) any later version.
+#
+# This library is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this library. If not, see <https://www.gnu.org/licenses/>.
+
+# With the goal of avoid deps. This is a mini version of needed part of
+# the https://github.com/platformdirs/platformdirs lib
+
+import os
+import sys
+from typing import Callable
+
+try:
+    import platformdirs
+
+    USE_SYSTEM_PLATFORMDIRS = True
+except ModuleNotFoundError:
+    USE_SYSTEM_PLATFORMDIRS = False
+
+
+def _linux_user_data_dir(appname: str) -> str:
+    """Return data directory tied to the user"""
+    path = os.environ.get("XDG_DATA_HOME", "")
+    if not path.strip():
+        path = os.path.expanduser("~/.local/share")
+    return os.path.join(path, appname)
+
+
+def _linux_site_data_dir(appname: str) -> str:
+    """Return data directory shared by users"""
+    path = os.environ.get("XDG_DATA_DIRS", "")
+    if not path.strip():
+        path = "/usr/share"
```

Comment:
The default value from the spec is /usr/local/share/:/usr/share/

Reply:
Fixed in d9125cd